index.html

```diff
@@ -1,75 +1,216 @@
 <!doctype html>
 
-<html lang="en">
+<html lang= "en">
+
+
+
 <head>
-  <meta charset="utf-8">
+    
+
+    <meta charset= "utf-8">
+  
+    <title> Sprint Challenge - Home  
+    </title>
+    
+    <link href= "https://fonts.googleapis.com/css?family=Roboto|Rubik" rel= "stylesheet">
+    <link rel= "stylesheet" href= "css/index.css">
 
-  <title>Sprint Challenge - Home</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+</head>  
+
 
-</head>
 
 <body>
-    <div class="container">
    
-        <section class="top-content">
-            <div class="text-container">
-                <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+    
+    <header>
+
+        
+        <div class= "icon">
+            <img src= "img/lambda-black.png"/>
+        </div>
+        
+        <nav class= "links">
+            
+            <div class= "h">
+                <a href= "index.html"> Home
+                </a>
             </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+            
+            <div class= "a">
+                <a href= "about.html"> About
+                </a>
             </div>
-        </section>
         
-        <section class="middle-content">
+            <div class= "p">
+                <a href= "#products"> Products
+                </a>
+            </div>
+        
+            <div class= "b">
+                <a href= "#blog"> Blog
+                </a>
+            </div>
+        
+            <div class= "c">
+                <a href= "#contact"> Contact
+                </a>
+            </div>
+          
+        </nav>
+    
+
+    </header>
+
+
+    <section>
```

Comment:
The header would be a better name for where the jumbo image goes, you can even just move this into the header if you like.